web/annotation_layer_builder.css

```diff
@@ -27,16 +27,6 @@
   }
 }
 
-[data-annotation-rotation="90"] {
-  transform: rotate(90deg) translateY(-100%);
-}
-[data-annotation-rotation="180"] {
-  transform: rotate(180deg) translate(-100%, -100%);
-}
-[data-annotation-rotation="270"] {
-  transform: rotate(270deg) translateX(-100%);
-}
-
```

Comment:
To ensure that moving this code won't cause any issues now, or in the future, I've submitted PR #15101 which should help avoid any problems.